src/NexusMods.App.UI/LeftMenu/Items/ApplyControlView.axaml.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.ReactiveUI;
+using ReactiveUI;
+
+namespace NexusMods.App.UI.LeftMenu.Items;
+
+public partial class ApplyControlView : ReactiveUserControl<IApplyControlViewModel>
+{
+    public ApplyControlView()
+    {
+        InitializeComponent();
+
+        this.WhenActivated(disposables =>
+        {
+            this.BindCommand(ViewModel, vm => vm.ApplyCommand, v => v.ApplyButton)
+                .DisposeWith(disposables);
+
+            this.OneWayBind(ViewModel, vm => vm.LaunchButtonViewModel, v => v.LaunchButtonView.ViewModel)
+                .DisposeWith(disposables);
+
+            this.WhenAnyValue(view => view.ViewModel!.CanApply)
+                .OnUI()
+                .BindToView(this, view => view.ApplyButton.IsVisible)
+                .DisposeWith(disposables);
+
+            this.WhenAnyValue(view => view.ViewModel!.IsApplying)
+                .OnUI()
+                .BindToView(this, view => view.InProgressBorder.IsVisible)
+                .DisposeWith(disposables);
+
+            this.WhenAnyValue(view => view.ViewModel!.IsApplying)
+                .Select(isApplying => !isApplying)
+                .OnUI()
+                .BindToView(this, view => view.LaunchButtonView.IsVisible)
+                .DisposeWith(disposables);
```

Comment:
Move these `OnUI` calls to the VM and use simple `OneWayBind` calls instead.

Reply:
I'm setting the values of the reactive properties from a task, should I use OnUi inside that task?

Reply:
`IsApplying` is the same as `ApplyCommand.IsExecuting`, is it not?

Reply:
It is in this implementation, but in the future the ApplyService will determine that, since there might be another loadout applying to the same GameInstallation

Reply:
Then just have the `IsApplying` property be set in the VM by subscribing to the `ApplyCommand.IsExecuting` property for now.